business/workloads.go

```diff
@@ -225,6 +240,21 @@ func (in *WorkloadService) getParsedLogs(namespace, name string, opts *core_v1.P
 
 		parsed, err := time.Parse(time.RFC3339, entry.Timestamp)
 		if err == nil {
+			if startTime == nil {
+				startTime = &parsed
+			}
+
+			if opts.Duration != nil {
+				if endTime == nil {
+					end := parsed.Add(*opts.Duration)
+					endTime = &end
```

Comment:
If you know of a better way to do this I'm happy to change. The error is that for some reason I can not do something like this `endTime = &parsed.Add(*opts.Duration)`, it fails on compilation.

Reply:
It's fine, that's normal for Go as you can't take the address of a funtion return.